src/image_copy/main.go

```diff
@@ -119,7 +119,16 @@ func mustHaveArchitecture(imageTag, platform, expectedArch string) {
 		log.Fatal("no binaries were found")
 	}
 	for _, file := range files {
-		out = execCmd("file", filepath.Join(tmpDir, file.Name()))
+		fp := filepath.Join(tmpDir, file.Name())
+		info, err := os.Lstat(fp)
+		if err != nil {
+			log.Fatal(err.Error())
+		}
+		if info.Mode()&os.ModeSymlink != 0 {
+			// skip symlinks
+			continue
```

Comment:
Should we be following and copying whatever the symlinks point to?

Reply:
I tried that, but it doesn't work. And even if it worked, the script would also fail because `cqlsh` is a Python file, not an actual binary.